bind/symtab.go

```diff
@@ -130,6 +130,10 @@ func (s symbol) isMap() bool {
 	return (s.kind & skMap) != 0
 }
 
+func (s symbol) isSequence() bool {
```

Comment:
I think `isPySequence` better conveys what piece of information we want to extract and its later use.